numba/tests/test_inplace.py

```diff
@@ -0,0 +1,23 @@
+from __future__ import print_function
+import numba.unittest_support as unittest
+from numba import types, utils, jit
+from numba.tests import usecases
+import numpy as np
+
+@jit(nopython=True)
+def add_5(x):
+    x += 5
+
+
+class TestInplaceAdd(unittest.TestCase):
```

Comment:
There's already a bunch of tests for operators in test_ufuncs (look for "Array operators"), so I think you should just add yours there.